fidesctl/fides_resources/dataset.yml

```diff
@@ -0,0 +1,239 @@
+dataset:
+  - fides_key: fidesctl
+    organization_fides_key: default_organization
+    name: fidesctl
+    description: "Fides Generated Description for Dataset: fidesctl"
```

Comment:
It would be cumbersome to fix them all, but is it a bad look to leave these auto-generated strings in this file?

Reply:
On one hand I think its vaguely ok because it shows we use our own tool, but I should put a more descriptive description in there for sure